game/scripts/npc/abilities/talents/undying_talent8.txt

```diff
@@ -0,0 +1,26 @@
+"DOTAAbilities"
+{
+  //=================================================================================================================
+  // Bonus Decay Damage
+  //=================================================================================================================
+  "special_bonus_unique_undying_8"
+  {
+    // General
+    //-------------------------------------------------------------------------------------------------------------
+    "ID"					""														// unique ID number for this ability.  Do not change this once established or it will invalidate collected stats.
+    "AbilityType"					"DOTA_ABILITY_TYPE_ATTRIBUTES"
+    "AbilityBehavior"				"DOTA_ABILITY_BEHAVIOR_PASSIVE"
+
+    // Special
+    //-------------------------------------------------------------------------------------------------------------
+    "AbilitySpecial"
+    {
+      "01"
+      {
+        "var_type"					"FIELD_INTEGER"
+        "value"				"110" //OAA
```

Comment:
Not sure if necessary. Valve added this talent so Undying could clear waves. Was your goal to make Undying a better farmer at lvl 10 or just doubling the bad number? In OAA str steal scales so I am pretty sure that this talent is overkill in OAA (buffing it could even make him op).